app/backend/app.py

```diff
@@ -604,10 +609,15 @@ async def setup_clients():
         hf_client = HuggingFaceClient(token=HUGGINGFACE_API_KEY)
 
     llm_clients: dict[str, LLMClient] = {"openai": openai_client, "hf": hf_client}
+    protection_config = PromptProtectionConfig()
+
+    if USE_INJECTION_PROTECTION:
+        protection_config.injection_protection.use = True
```

Comment:
This feels a bit wordy. Can't we do something like this?

```suggestion
    protection_config = PromptProtectionConfig(use=USE_INJECTION_PROTECTION)
```

Reply:
Oh no, we can't due to the way these dataclasses are modelled. I'll have to think a bit about this but it does look weird exposing an API like this.